### PR TITLE
Add CLI for Loklak API

### PR DESCRIPTION
**Pull request by sevazhidkov**

### CODE REVIEW

File: loklak.py [hunk 1]
```diff
@@ -234,4 +234,45 @@ def main():
...
+        args = parser.parse_args()
+        loklak = Loklak()
+        if args.search:
+            pprint(loklak.search(
+                 input('Enter query: '),
+                 input('Enter since: '),
+                 input('Enter until: '),
+                 input('Enter fromUser: ')
+            ))
+        elif args.status:
```
Comment (sudheesh001) [inline loklak.py]: Can we have a basic search system implemented without asking the user to take inputs. 
`loklak --search <query list to search for>`

This is a good attempt, but the aim is to limit the interaction and provide a single shot method to run a query. Can you modify this please ? The same goes for the user CLI

**Comment by sevazhidkov** [comment]
I'm done, but i don't know why tests is failing, i didn't edit anything in Loklak class.

**Comment by sudheesh001** [comment]
@sevazhidkov Looks great but its still not a command line interface. I need to use `python loklak.py -hl` etc..,
It needs to be `loklak -hl` for example without the `python`

**Comment by sevazhidkov** [comment]
Done.

**Comment by sudheesh001** [comment]
Where is the `main()` function ? This will throw an error
```
ImportError: <module 'loklak' lib/python2.7/site-packages/python_loklak_api-1.0-py2.7.egg/loklak.pyc'> has no 'main' attribute
```

**Comment by sevazhidkov** [comment]
My bad. Fixed.

File: bin/loklak [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import argparse
+import loklak
+
```
Comment (sudheesh001) [inline bin/loklak]: You forgot to add `from pprint import pprint`

**Comment by sevazhidkov** [comment]
Done

**Comment by sudheesh001** [comment]
@sevazhidkov Great work ! The command line now works brilliantly.

**Pull request merged**